deploy/charts/rook-ceph/values.yaml

```diff
@@ -529,6 +537,25 @@ csi:
     # @default -- labels listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology)
     crushLocationLabels:
 
+  # -- specify cephFSAttachRequired field to instruct Kubernetes to skip any attach operation altogether
+  # for CephFS PVC. More details at https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object
+  # If cephFSAttachRequired is set to false it skips the volume attachments and makes the creation
+  # of pods using the CephFS PVC fast but it will have consequences of mounting RWO volumes on multiple nodes.
+  # More details at https://github.com/kubernetes/kubernetes/issues/103305
+  cephFSAttachRequired: true
+  # -- specify rbdAttachRequired field to instruct Kubernetes to skip any attach operation altogether
+  # for RBD PVC. More details at https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object
+  # If rbdAttachRequired is set to false it skips the volume attachments and makes the creation
+  # of pods using the RBD PVC fast but it will have consequences of mounting RWO volumes on multiple nodes
+  # More details at https://github.com/kubernetes/kubernetes/issues/103305
+  rbdAttachRequired: true
+  # -- specify nfsAttachRequired field to instruct Kubernetes to skip any attach operation altogether
+  # for NFS PVC. More details at https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object
+  # If nfsAttachRequired is set to false it skips the volume attachments and makes the creation
+  # of pods using the NFS PVC fast but it will have consequences of mounting RWO volumes on multiple nodes.
+  # More details at https://github.com/kubernetes/kubernetes/issues/103305
+  nfsAttachRequired: true
```

Comment:
the default value is `false` for nfs, right?

Reply:
No, it was changed to true in the first commit to avoid the mounting RWO on multiple nodes issue.

Reply:
Isn't RWO rare for both cephfs and nfs? In any case, it's unfortunate that the attacher can't have a different setting for different volume types.

Reply:
Yes based on the usecase yes but as we are advertising the caps we need to support it even if it's rare to avoid corner cases. once kubernetes enforces proper check we can remove it completely